docs/en/manifest.json

```diff
@@ -9,7 +9,8 @@
       "contributing": "Contributing",
       "issue-test-case": "Issue with Test Case",
       "intro/semver": "Versioning",
-      "license": "LICENSE"
+      "license": "LICENSE",
+      "integrations": "Integrations"
```

Comment:
Perhaps it's worth adding this item to the menu above the 'License' item? @roxblnfk What do you think?